app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -114,7 +114,8 @@ function initializePodcastPlayback() {
     return (
       navigator.userAgent === 'DEV-Native-android' &&
       typeof AndroidBridge !== 'undefined' &&
-      AndroidBridge !== null
+      AndroidBridge !== null &&
```

Comment:
@maestromac I kept the double check against `undefined` & `null` because of the linter complaining about using a single `==` comparison. If you rather only do one I think we can scrap the `null` comparison instead

Reply:
Ah, it's no biggie 👍